src/diagnostic/detail/types/fluid.h

```diff
@@ -50,6 +55,31 @@ class FluidDiagnosticWriter : public H5TypeWriter<HighFiveDiagnostic>
 };
 
 
+
+template<typename HighFiveDiagnostic>
+void FluidDiagnosticWriter<HighFiveDiagnostic>::createFiles(DiagnosticProperties& diagnostic)
+{
+    auto& hi5 = this->hi5_;
+
+    auto checkCreate = [&](auto& tree, auto var) {
+        auto active = tree + var;
+        bool b      = diagnostic.quantity == active;
+        if (b && !fileData.count(diagnostic.quantity))
```

Comment:
we can now use `and` instead of `&&` I find it easier to read

Reply:
k